src/stdlib_strings_format_string.fypp

```diff
@@ -29,13 +29,10 @@ contains
         !! Format ${type}$ variable as character sequence
         character(len=buffer_len) :: buffer
```

Comment:
`buffer` is no longer needed. It can be removed.

Reply:
Good catch.